tools/setup_perf_cluster.sh

```diff
@@ -163,56 +163,66 @@ function install_istio_cache_busting_rule() {
   Execute $ISTIOCTL create -n $ISTIO_NAMESPACE -f $FNAME
 }
 
-function get_fortio_k8s_ip() {
-  FORTIO_K8S_IP=$(kubectl -n $FORTIO_NAMESPACE get svc -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
-  echo "+++ In k8s fortio external ip: http://$FORTIO_K8S_IP:8080/fortio/"
+function get_fortio1_k8s_ip() {
+  FORTIO1_K8S_IP=$(kubectl -n $FORTIO_NAMESPACE get svc -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
+  echo "+++ In k8s fortio1 non-istio external service: http://$FORTIO1_K8S_IP:8080/fortio/"
 }
 
-# Doesn't work somehow...
-function setup_non_istio_ingress2() {
-  cat <<_EOF_ | kubectl apply -n fortio -f -
+function get_fortio2_k8s_ip() {
+  FORTIO2_K8S_IP=$(kubectl -n $FORTIO_NAMESPACE get svc -o jsonpath='{.items[1].status.loadBalancer.ingress[0].ip}')
+  echo "+++ In k8s fortio2 non-istio external service: http://$FORTIO2_K8S_IP:8080/fortio/"
+}
+
+function setup_non_istio_ingress() {
+  cat <<_EOF_ | kubectl apply -n "$FORTIO_NAMESPACE" -f -
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: fortio-ingress2
+  name: fortio-ingress
 spec:
-  rules:
-  - http:
-      paths:
-       - path: /fortio1
```

Comment:
ah I see what I wanted here ... I was trying to use an (istio like) ingress  spec to get the same setup

it'd be nice to make it work rather than just drop it ?

Reply:
OK. I was thinking that the ingress spec for fortio1 and fortio2 should be identical. Since the fortio1 spec worked, I used that as the reference.

Can we refactor the ing specs in a future PR?

Reply:
this one is supposed to be fortio1 and doesn't work in term of prefix - but yes I guess ingress setup we can change later